server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadEndpoint.feature

```diff
@@ -74,9 +74,14 @@ Feature: Download endpoint
     When "userb@domain.tld" downloads "a1"
     Then the user should be authorized
 
-  @Ignore
-  Scenario: A user should have acess to an inlined attachment
+  Scenario: A user should have access to an inlined attachment
     Given "usera@domain.tld" is connected
     And "usera@domain.tld" mailbox "INBOX" contains a message "m2" with an inlined attachment "ia1"
     When "usera@domain.tld" downloads "ia1"
     Then the user should be authorized
+
+  Scenario: A user should have access to multiple same inlined attachments
+    Given "usera@domain.tld" is connected
+    And "usera@domain.tld" mailbox "INBOX" contains a message "m2" with multiple same inlined attachments "ia1"
+    When "usera@domain.tld" downloads "ia1"
```

Comment:
Yes, it had:

> When "usera@domain.tld" downloads "ia1"
    And "usera@domain.tld" downloads "ia1"
    Then the user should be authorized

Is that your meaning?